etl/steps/grapher/ihme_gbd/2019/gbd_cause_omms.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
You can either have them in JSON and then write meta code for constructing conditions in `omm_metrics` functions or just write code directly. Up to you, I guess it also depends on number of OMMs you'll have.